spec/services/controlled_work_document_value_mapping_service_spec.rb

```diff
@@ -82,6 +82,20 @@
       }
       expect(result).to include(representative_sample)
     end
+
+    it "can successfully populate CW5 form fields" do
+      mappings = YAML.load_file(Rails.root.join("app/lib/controlled_work_mappings/cw5.yml")).map(&:with_indifferent_access)
```

Comment:
as this form includes SMOD we should probably also test that SMOD values are populating